exasol/slc/tool/commands/run_db_tests.py

```diff
@@ -149,6 +150,13 @@
 @add_options(system_options)
 @add_options(luigi_logging_options)
 @add_options(export_options)
+@click.option(
+    "--accelerator",
+    type=click.Choice([a.value for a in Accelerator]),
```

Comment:
nit: could add a method to `Accellerator` class which provides this list & can use in 153 & 158.

Reply:
Done